bin/heudiconv

```diff
@@ -935,6 +946,46 @@ def convert(items, symlink=True, converter=None,
     shutil.rmtree(tmpdir)
 
 
+def get_formatted_scans_key_row(item, outname_bids_files):
```

Comment:
please add docstrings to those especially with the Return description... you know how it is easy to figure things out later on ;)